src/platform/site-wide/announcements/components/AfghanistanPromoBanner.jsx

```diff
@@ -6,6 +6,7 @@ import PromoBanner, {
 
 export default ({ dismiss }) => (
   <PromoBanner
+    role="alert"
```

Comment:
I don't think we need this if the default role prop is `"alert"` if you want to omit it 🙂

Reply:
We will need to bump the package.json version here for the component-library for our PromoBanners to get the new role prop functionality 💯